awscli/autoprompt/prompttoolkit.py

```diff
@@ -131,7 +134,7 @@ def _set_app_defaults(self, app):
         app.debug = False
 
     def _update_doc_window_contents(self, *args):
-        parsed_args = self._completion_source.parse(
+        parsed_args = self._parser.parse(
```

Comment:
We should probably remove the public `parse()` method on the `AutoCompleter` class since that is not really used anymore so that we do not have any unnecessary interfaces.